src/consensus/aft/impl/state.h

```diff
@@ -98,7 +98,8 @@ namespace aft
       my_node_id(my_node_id_),
       current_view(0),
       last_idx(0),
-      commit_idx(0)
+      commit_idx(0),
+      new_view_idx(0)
```

Comment:
What's this value? Is it what we think the next view should be?

Reply:
This is used by the primary to track at which index the last NewViewConfirmation is stored so that it can send a hint when replicating append entries